src/main/java/io/patriot_framework/virtual_smart_home/route/FireplaceModel.java

```diff
@@ -0,0 +1,34 @@
+package io.patriot_framework.virtual_smart_home.route;
+
+import io.patriot_framework.virtual_smart_home.house.device.Fireplace;
+
+class FireplaceModel {
+
+    private String label;
+    private boolean enabled;
+
+    public String getLabel() {
+        return label;
+    }
+
+    public void setLabel(String label) {
+        this.label = label;
+    }
+
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    public Fireplace toFireplace() {
+        Fireplace fireplace = new Fireplace(label);
+        if (enabled) {
+            fireplace.fireUp();
+        }
+
+        return fireplace;
+    }
```

Comment:
This is just a model class for parsing JSON from requests. This code was meant to create a `Fireplace` out of parsed data, but the `Fireplace` itself has only one constructor which takes only the `label` parameter in `String` format, so I had to make the `Fireplace` as it was sent in the request and therefore if the `Fireplace` had the `enabled` value set to `true`, I had to create already enabled `Fireplace` from it. This was the easiest way so far to do that but of course it can be modified in the future.